ldk/javascript/examples/consolidate-loop/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "javascript-loop-example",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "webpack --entry ./index.js --config ./node_modules/@oliveai/ldk/dist/webpack/config.js"
+  },
+  "author": "LDK Team",
+  "license": "ISC",
+  "ldk": {
+    "permissions": {
+      "clipboard": {},
+      "keyboard": {},
+      "whisper": {},
+      "ui": {},
+      "filesystem": {
+        "pathGlobs": [
+          {
+            "value": "./PatientInfo.txt"
```

Comment:
I believe we dont need to specify glob permissions for the  ./ any more. As long as you have "filesystem": {} you should be good to create './PatientInfo.txt'. This change was made earlier this week.